### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by DanielSantaRosaZup** [sonnemaf/ReflectionIT.Analyzer#5]
Hi there,

Where can I find this project documentation?

**Comment by kkm000** [comment]
Yes, yes, YES, please!!! A list of analysers is not enough without at least a short paragraph what they're doing! VS highlights a lot of [non]-issues, so I'm wary of installing analysers, as it's hard to track a fiadnostics back to the analyser that got angry. Just installing them and intuitively figuring out what they do is in fact complicated.
